pkg/skr/backupschedule/evaluateNextRun_test.go

```diff
@@ -126,7 +126,12 @@ func (suite *evaluateNextRunSuite) TestWhenNextRunTimeIsNotDueYet() {
 	err, _ = evaluateNextRun(ctx, state)
 
 	//validate expected return values
-	suite.Equal(composed.StopWithRequeueDelay(offset), err)
+	result, err := composed.HandleWithoutLogging(err, ctx)
+	delay := result.RequeueAfter
+	suite.Nil(err)
+	suite.NotNil(delay)
+	suite.Greater(delay, time.Duration(0))
```

Comment:
Why allow so big gap? Give it some relative tolerance, like 5%. 
```suggestion
	suite.Greater(delay, time.Duration(int64(0.95 * float64(offset))))
```